web-gui/src/app/util/rx-stomp.ts

```diff
@@ -1,80 +1 @@
-import {Client, CompatClient, Frame, IMessage, Stomp, StompSubscription} from '@stomp/stompjs';
-import {Observable, Subscriber} from 'rxjs';
-import * as SockJS from 'sockjs-client';
 import {Subscription} from 'stompjs';
```

Comment:
@Zitrone44  when this file is removed, several compiler errors occur (see [here](https://github.com/thm-mni-ii/feedbacksystem/runs/4425128466?check_suite_focus=true#step:7:122)) which have nothing to do with it, any idea why?

Reply:
I have found the error/a solution (see ee429f7642cd82a3ce1156d9554dac08ea94db9f)